.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Consider adding this file to `.gitignore` cause this is your editor configuration :)

Reply:
@lilka I saw that you added file to gitignore, but this file is still tracked by GitHub cause it was commited before adding.

Please consider [this article](https://vershd.io/git-ignore-already-tracked-files)